pkgs/desktops/cinnamon/nemo/default.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub, glib, gobject-introspection, meson, ninja, pkgconfig, stdenv, wrapGAppsHook, libxml2, cmake, gtk3, libnotify, cinnamon-desktop, xapps, libexif, exempi, intltool }:
+
+stdenv.mkDerivation rec {
+  pname = "nemo";
+  version = "4.4.1";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "0sskq0rssxvna937md446x1489hkhxys1zq03hvl8asjqa259w2q";
+  };
+
+  patches = [ ./0001-patch-datadir.patch ];
+
+  preInstallPhases = "exportDestdir";
```

Comment:
This should be a list (preparation for `_structuredAttrs`). Or just use `preInstall`.